README.md

```diff
@@ -2,7 +2,25 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ## Getting Started
 
-First, run the development server:
+First, clone the repo and cd into your directory
+
+```bash
+git clone https://github.com/fintual-oss/paraffin-front.git && cd paraffin-front
+```
+
+Create a `.env.local` file and ask someone for the ENV variables
+
+```bash
+touch .env.local
+```
+
+Install dependencies
+
+```bash
+yarn install
+```
+
+Run the development server
 
 ```bash
 npm run dev
```

Comment:
It should be `yarn dev`, as we are using Yarn.

Reply:
Removed